website/docs/r/pi_network_interface.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_pi_network_interface"
```

Comment:
```suggestion
page_title: "IBM : pi_network_interface"
```

Reply:
@yussufsh Shouldn't this be same as the title? `# ibm_pi_network_interface`
That is what other services do from taking a look at most of them.